TACC/TACC-Docs
# chore: Dockerfile use `--without dev` not `--no-dev`

## Overview

Use `--without dev`, because [`--no-dev` is deprecated](https://python-poetry.org/docs/master/cli/#options-2).

## Related

N/A

## Changes

* **changed** Dockerfile `poetry install` command

## Testing

1. Build site via `make build`.
2. Verify successful build.
3. Serve site via `make start`.
4. Verify successful serve.
5. [Deploy site.](https://confluence.tacc.utexas.edu/pages/viewpage.action?pageId=311559865#HowToBuild&DeployaDocsWebsite-3.Deploy)
6. Verify successful deploy.
7. Load site.
6. Verify successful load.

## UI

Skipped.
